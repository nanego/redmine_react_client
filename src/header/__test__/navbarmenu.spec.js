import React from 'react';
import NavBarMenu from '../navbarmenu';
import renderer from 'react-test-renderer';

it('should render NavBarMenu', () => {
  let applyFiltersChanges = function(){};
  let updateSelectedFilters = function(){};
  let selected_filters_as_text = function(){};
  let replaceSelectedFilters = function(){};
  let current_filters = {};
  let selected_filters = {};
  let dirty_filters = {};
  const component = renderer.create(
      <NavBarMenu current_filters={current_filters}
                  selected_filters={selected_filters}
                  selected_filters_as_text={selected_filters_as_text}
                  applyFiltersChanges={applyFiltersChanges}
                  updateSelectedFilters={updateSelectedFilters}
                  replaceSelectedFilters={replaceSelectedFilters}
                  dirty_filters={dirty_filters}
      />
  );
  let navbar = component.toJSON();
  expect(navbar).toMatchSnapshot();
});