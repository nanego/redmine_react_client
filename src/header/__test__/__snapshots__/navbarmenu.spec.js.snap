exports[`test should render CustomQueries 1`] = `
<div
  className="ui secondary vertical menu">
  <div
    aria-busy={undefined}
    aria-disabled={undefined}
    aria-expanded={false}
    aria-multiselectable={undefined}
    className="ui item dropdown"
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
    role="listbox"
    tabIndex={0}>
    <div
      className="text">
      Mes demandes par priorité
    </div>
    <div
      className="menu transition" />
  </div>
  <div
    aria-busy={undefined}
    aria-disabled={undefined}
    aria-expanded={false}
    aria-multiselectable={undefined}
    className="ui item dropdown"
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
    role="listbox"
    tabIndex={0}>
    <div
      className="text">
      Demandes surveillées
    </div>
    <div
      className="menu transition" />
  </div>
  <div
    aria-busy={undefined}
    aria-disabled={undefined}
    aria-expanded={false}
    aria-multiselectable={undefined}
    className="ui item dropdown"
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
    role="listbox"
    tabIndex={0}>
    <div
      className="text">
      Traité sans activité récente
    </div>
    <div
      className="menu transition" />
  </div>
  <div
    aria-busy={undefined}
    aria-disabled={undefined}
    aria-expanded={false}
    aria-multiselectable={undefined}
    className="ui item dropdown"
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
    role="listbox"
    tabIndex={0}>
    <div
      className="text">
      Recherche avancée
    </div>
    <i
      aria-hidden="true"
      className="dropdown icon" />
    <div
      className="menu transition" />
  </div>
</div>
`;
